Source/WebInspectorUI/UserInterface/Views/FontVariationDetailsSectionRow.css

```diff
@@ -0,0 +1,87 @@
+.details-section > .content > .group > .row.font-variation {
+    display: grid;
+    gap: 6px;
+    grid-template-columns: 36px 1fr minmax(max-content, 36px);
+    padding-block: 5px;
+    padding-inline: 20px;
```

Comment:
Should these values match other types of rows (I suspect the answer is yes)? If so, why do we need to explicitly define them here, since the `.row` should already have them? Or if we are adding padding to existing padding, some variables and `calc` to make sure we add a consistent amount to the default padding so that if the other padding changes this follows suit?

Reply:
A look at `DetailsSection.css` shows significant variance between padding for different types of rows.
The contents of `DetailsSectionRow` instances are very diverse: from read-only lists (Fonts), to tables (Node panel), to checkboxes and labels (Flex + Grid), etc. 

The variation axis control in Fonts warrants special padding to aid visual separation.

`padding-left: 12px` seems to be most prevalent among rows to somewhat align their contents with the arrow toggle icon of the section header. I'll use that value, but I'll keep a wide `padding-right` to avoid overlap with the scrollbar in its expanded state.

Here's 12px uniform inline padding where the axis get partially covered:
<img width="321" alt="Screenshot 2023-01-05 at 12 02 23 PM" src="https://user-images.githubusercontent.com/105809798/210777524-c9d38f18-9e8c-4e5d-b86a-136c370577b1.png">